src/data/finsupp.lean

```diff
@@ -1156,6 +1156,9 @@ def to_semimodule {R:semiring γ} [add_comm_monoid β] [semimodule γ β] : semi
 def to_module {R:ring γ} [add_comm_group β] [module γ β] : module γ (α →₀ β) :=
 { ..to_semimodule α β }
 
+def to_vector_space {R:discrete_field γ} [add_comm_group β] [vector_space γ β] : vector_space γ (α →₀ β) :=
```

Comment:
This instance and the ones above it can be actual instances, and the `R` can be a nameless instance implicit argument like the others now. This is a holdover from before the module refactor

Reply:
Done.

Reply:
@benti, it helps the reviewing process if you click "resolve conversation" when appropriate.